src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -92,13 +93,26 @@ def generate_WDB_1_default_config(self, filename):
 
     conditionals = [conditionals_b]
 
-    # Create the actual config file
+    pal_db_relative_file_path = os.path.join('testcases', 'testdata', 'pal_db', 'pal_db_record.json')
+
+    # Create the pal_db record file consistent with CBSDs and PAL records.
+    with open(os.path.join(pal_db_relative_file_path), 'w+') as file_handle:
```

Comment:
Why do you need another call to os.path.join() here?

Reply:
Not required, hence removed.